pkg/kapp/cmd/app/app_flags.go

```diff
@@ -12,10 +12,12 @@ import (
 type Flags struct {
 	NamespaceFlags cmdcore.NamespaceFlags
 	Name           string
+	AppNamespace   string
 }
 
 func (s *Flags) Set(cmd *cobra.Command, flagsFactory cmdcore.FlagsFactory) {
 	s.NamespaceFlags.Set(cmd, flagsFactory)
 
 	cmd.Flags().StringVarP(&s.Name, "app", "a", s.Name, "Set app name (or label selector) (format: name, label:key=val, !key)")
+	cmd.Flags().StringVar(&s.AppNamespace, "app-namespace", s.AppNamespace, "Set app namespace (to store app state)")
```

Comment:
Should we indicate that this defaults to `-n` otherwise?

Reply:
At first I thought of doing that too, but if we say `-n`, then that is option too, and we will have to mention everything from `-n`. I skipped it thinking that `-n` description already says that part.